src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -30048,6 +30287,381 @@ .maxstack  3
                 """);
         }
 
+        [Fact]
+        public void SingleSpread_WellKnownMemberMissing()
+        {
+            var source = """
+                class C
+                {
+                    static void Main()
+                    {
+                        int[] arr = [1, 2, 3];
+                        arr.Report();
+                        int[] arr1 = [..arr];
+                        arr1.Report();
+                    }
+                }
+                """;
+
+            var comp = CreateCompilation([source, s_collectionExtensions], targetFramework: TargetFramework.Net80, options: TestOptions.ReleaseExe);
+            comp.VerifyEmitDiagnostics();
```

Comment:
>VerifyEmitDiagnostics [](http://example.com/codeflow?start=17&length=21)

Minor, non-blocking suggestion: Consider calling ```VerifyDiagnoctics``` on result of ```CompileAndVerify``` instead. This should eliminate a redundant emit and will make the test faster a bit.